packages/labs/testing/src/lib/fixtures/csrFixture.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2022 Google LLC
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+import {render} from 'lit';
+import type {LitElement, TemplateResult} from 'lit';
+import type {FixtureOption} from './fixtureOption.js';
+
+/**
+ * Renders the provided lit-html template with a Lit element client-side.
+ *
+ * @param {TemplateResult} template - lit-html template. Must contain a single
```

Comment:
I don't think we want the `{types}` in the jsdoc, since the types should get correctly extracted by vscode and other tools directly from the typescript syntax, and that's a better source of truth.